dll/win32/oleaut32/connpt.c

```diff
@@ -22,9 +22,22 @@
  * comments just above that function for information.
  */
 
-#include "precomp.h"
-
```

Comment:
This will break building with precompiled headers?

Reply:
Wine doesn't have precompiled headers. I think the introduction of precomp.h has been a ReactOS-specific change just for the sake of faster builds. The downsides are obviously more diffs to upstream.
If it was me, I would always trade a slightly longer build for less diffs to upstream code.

Reply:
I absolutely agree. Let's screw these few seconds of builds time we can gain and go back to (almost) unmodified 3rd party code. With these include changes conflicts when rebasing are almost certain, and usually causing a build breakage that has to be manually investigated and resolved. Not worth the effort.

Reply:
In fact it turned out that with PCH there is no difference, since PCHs are force included on the command line, so with PCH it doesn't matter what's included here. And without PCH the original code builds faster with smaller object files. So the changes are not only unnecessary, they are counter productive.

Reply:
Just in case more of us don't remember what the headers work was about, and how it was *completely* unrelated to PCH, here's a quick picture of back then:

1. We were using, shamefully, the stabs debug format, and that was crippling our builds to the point that just the *initial* headers cleanup significantly shrank the dependency tracking data and the compiled objects size, speeding up the whole build process, especially incremental builds, as a result. We're talking ~27% smaller builds at that time without a single *functional* change needed.

2. Our PCH approach back then was different from the one was have now where we force include the PCH. We didn't even have PCH for MSVC builds and we didn't even have Clang builds to begin with, let alone have PCH support for them.

3. Eventually I got us switched to the compressed dwarf debug format, and introduced the force included PCH approach and got us Ninja builds, support for MSVC and Clang... etc. *That* is why we ended up now with headers blocks and the current, unified PCH approach among all supported compilers.

Hopefully you will realize from the above that these changes were *very* productive and necessary, and maybe appreciate the amount of work done at that time and after, that led to the *luxury* we have now of declaring them not only unnecessary but also counter productive :)